<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camera_switcher</class>
 <widget class="QMainWindow" name="camera_switcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Switcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>220</width>
         <height>142</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="button_layout">
          <item>
           <widget class="QPushButton" name="persp_bt">
            <property name="text">
             <string>persp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="top_bt">
            <property name="text">
             <string>top</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="front_bt">
            <property name="text">
             <string>front</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="side_bt">
            <property name="text">
             <string>side</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>238</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="create_menu_bar">
    <property name="title">
     <string>Create...</string>
    </property>
    <addaction name="action_create_camera"/>
   </widget>
   <addaction name="create_menu_bar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_create_camera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
